src/adb/command/host/disconnect.coffee

```diff
@@ -5,7 +5,8 @@ class DisconnectCommand extends Command
   # Possible replies:
   # "No such device 192.168.2.2:5555"
   # ""
-  RE_OK = /^$/
+  # disconnected host:port
+  RE_OK = /^disconnected (?:[0-9]{1,3}\.){3}[0-9]{1,3}\:?([0-9]{1,5})|(^$)/
```

Comment:
I don't want to limit this to IP addresses, because that's not always the case. For example, in our STF deployments the connectable addresses have been something like `nexus-5-7abc3884.stf.example.org`.

Maybe just `/^disconnected |^$/` would be good enough?

Reply:
I agree. Thanks